PhysicsTools/PatAlgos/python/recoLayer0/bTagging_cff.py

```diff
@@ -338,3 +340,21 @@
 for disc in _pfParticleTransformerAK4JetTagsMetaDiscrs:
     supportedMetaDiscr[disc] = _pfParticleTransformerAK4JetTagsProbs
 # -----------------------------------
+
+# -----------------------------------
+# setup UnifiedParticleTransformer AK4
+from RecoBTag.ONNXRuntime.pfUnifiedParticleTransformerAK4_cff import _pfUnifiedParticleTransformerAK4JetTagsProbs, _pfUnifiedParticleTransformerAK4JetTagsMetaDiscrs
+# update supportedBtagDiscr
+for disc in _pfUnifiedParticleTransformerAK4JetTagsProbs + _pfUnifiedParticleTransformerAK4JetTagsMetaDiscrs:
+    supportedBtagDiscr[disc] = [["pfUnifiedParticleTransformerAK4TagInfos"]]
+# update supportedMetaDiscr
+for disc in _pfUnifiedParticleTransformerAK4JetTagsMetaDiscrs:
+    supportedMetaDiscr[disc] = _pfUnifiedParticleTransformerAK4JetTagsProbs
+# -----------------------------------
+
+# -----------------------------------
+# setup Negative ParticleTransformer AK4
```

Comment:
The negative ParticleTransformer AK4 is actually added in another PR: https://github.com/cms-sw/cmssw/pull/43485 and, since we do not have `_pfNegativeParticleTransformerAK4JetTagsProbs`, this would cause errors. I suggest we remove those lines

Reply:
Done in cc8a12e27a0e89b7117a3d113e49f941e767c4b4